content/en/resources/_index.md

```diff
@@ -1,7 +1,10 @@
 ---
 title: Resources
 menu: { main: { weight: 50 } }
-description: Learn more from our resources
+description:
 cascade:
   type: docs
 ---
+
+Find curated selections of videos, press coverage, and publications designed to
```

Comment:
You mention "press coverage" here.
Should there be a Press page under resources?